_plugins/jekyll-figurify.rb

```diff
@@ -0,0 +1,67 @@
+require 'jekyll'
+
+module Jekyll
+  class Figurify < Jekyll::Generator
+
+    safe true
+
+    def initialize(config)
+      @config = config['figurify'] ||= {}
+    end
+
+    def generate(site)
+      puts "\n      Applying figurify plugin ✨"
```

Comment:
@bebatut you should get rid of this line. It was fun but now it makes me crazy and I cannot see it anymore 😛